app/models/bot/alegre.rb

```diff
@@ -197,8 +197,8 @@ def self.get_number_of_words(text)
 
   def self.get_items_from_similar_text(team_id, text, fields = nil, threshold = nil, models = nil, fuzzy = false)
     team_ids = [team_id].flatten
-    if text.blank?
-      Rails.logger.info("[Alegre Bot] get_items_from_similar_text returning early due to blank text #{text}")
+    if text.blank? || /[a-z\-]+-[0-9\-]+/ =~ text
```

Comment:
I saw this regular expression being used in different places - so, if we're using it more than once, I suggest we save it to a constant, like we do with these constants: https://github.com/meedan/check-api/blob/develop/app/models/bot/alegre.rb#L12-L22. This way, if we need to change, there is no risk that we change in one place and not in another.

Also, I think we want exact matches, right? Like, "A very long title that contains this-123" would be valid according to this regular expression, but really we just wanted to match if it was "this-123", correct? Then, the regular expression should be `/^[a-z\-]+-[0-9\-]+$/`.

Reply:
thats true, thanks for the fix.